corehq/apps/app_manager/static/app_manager/json/v1/commcare-app-settings.yaml

```diff
@@ -38,6 +38,12 @@
   since: {qwerty: '2.6', 'custom-keys': '1.3'}
   requires: "{hq.platform}='nokia/s40'||{hq.platform}='nokia/s60'"
 
+- id: practice_mobile_worker_id
+  name: 'Practice Mobile Worker'
+  widget: select
+  since: '2.30'
+  privilege: 'practice_mobile_workers'
```

Comment:
could do with a description: "Select the mobile worker to use as a practice user for this application"